src/test/java/org/springframework/data/couchbase/domain/NaiveAuditorAware.java

```diff
@@ -0,0 +1,23 @@
+package org.springframework.data.couchbase.domain;
+
+import org.springframework.data.domain.AuditorAware;
+
+import java.util.Optional;
+
+//import org.springframework.security.core.Authentication;
```

Comment:
unused cleanup and license header

Reply:
Those commented imports are to show the classes that would be used when making a proper class to fetch the current username.  I've added a comment indicating that.